docker/caddy/Caddyfile

```diff
@@ -11,20 +11,6 @@ log
 
 route {
     root * /srv/app/public
-    mercure {
```

Comment:
It's perfectly fine to remove Mercure, but for the record it is supported by Laravel: https://github.com/mvanduijker/laravel-mercure-broadcaster